iceoryx_hoofs/include/iceoryx_hoofs/internal/posix_wrapper/shared_memory_object/shared_memory.hpp

```diff
@@ -52,6 +52,9 @@ enum class SharedMemoryError
     UNKNOWN_ERROR
 };
 
+/// @brief Shared memory file descriptor type
+using sharedMemoryHandle_t = int;
```

Comment:
I think we did not specified this explicitly and type aliases are currently a mess but I think we did not yet use lower camel case them. Could you please change this to `using SharedMemoryHandle_t = int;` or `using shared_memory_handle_t = int;` or `using shm_handle_t = int;`?

Reply:
Oh, of course. Changed now to `shm_handle_t`.